source/virtualBuffers/webKit.py

```diff
@@ -16,9 +17,27 @@
 import textInfos
 import NVDAHelper
 
+
+def _getNormalizedDescriptionAttrs(attrs) -> typing.Dict[str, typing.Any]:

```

Comment:
With what WebKit controls  this has been tested? The only one known to me is Itunes Store controls in Itunes and I find it hard to believe that `aria-description` is used there. If the intent was to provide different implementations for Chromium based browsers and Firefox this code should be placed in `NVDAObjects.IAccessible.chromium.ChromeVBuf` instead.

Reply:
This has been addressed.